js/modulestest/runtime.go

```diff
@@ -0,0 +1,51 @@
+// Package modulestest contains helpers to test js modules
+package modulestest
```

Comment:
hmm this seems like it should be in the `testutils` package (https://github.com/grafana/k6/tree/master/lib/testutils) :thinking: on the other hand, I wouldn't want for even a `lib` sub-package to depend on `js/`... :confused: so I wonder if we shouldn't move `testutils` to a top-level package and have this as a sub-package of it?

Reply:
Maybe but given the breaking change that will be (for both packages) this seems like a thing that might be nice but will have fairly low impact.

I kind of like the fact that we have two diffrent test helper packages that are more under the place where they should be used/the package they should help with. Intead of just having a global "all test helpers go here" kind of variant.